### PR TITLE
Small clean ups

### PR DESCRIPTION
**Pull request by Danielhiversen**

### CODE REVIEW

File: keras/utils/data_utils.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def dl_progress(count, block_size, total_size):
             if os.path.exists(fpath):
                 os.remove(fpath)
             raise
-        progbar = None
+        progress_tracker.progbar = None
```
Comment (Danielhiversen) [inline keras/utils/data_utils.py]: Is it necessary to reset the progbar to None, or should this line just be removed?

Comment (ahundt) [inline keras/utils/data_utils.py]: whoops! Thanks for the fix. I'd lean towards yes, but I don't know the details of how locally declared python classes work between calls.

**Comment by ahundt** [comment]
LGTM

**Comment by fchollet** [comment]
@ahundt to approve changes in a PR, you can use the GH review tools.

**Comment by ahundt** [comment]
ah, forgot the review menu vs regular post worked differently, thanks!

**Pull request merged**